src/index.html

```diff
@@ -2,11 +2,434 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header" id="home">
+      <a href="#home" class="header__home header__home--margin">
+        <img
+          src="images/logo.png"
+          alt="moyo_icon"
+        >
+      </a>
```

Comment:
```suggestion
      <a href="#home" class="header__home ">
        <img
          src="images/logo.png"
          alt="moyo_icon"
        >
      </a>
```
You can apply margins directly to BEM elements

Reply:
Fixi it everywhere

Reply:
Done